company-kata-solutions/src/test/java/org/eclipse/collections/companykata/Exercise6Test.java

```diff
@@ -126,6 +145,11 @@ public void supplierNamesAsTildeDelimitedString()
     @Test
     public void deliverOrdersToLondon()
     {
+        this.company.getCustomers()
+                .asLazy()
+                .selectWith((Predicate2<Customer, String>) (customer, city) -> customer.getCity().equals(city),"London")
+                .flatCollect(Customer::getOrders)
```

Comment:
Hi @nsardana-bny, please create a method on `Customer` if it doesn't exist named `livesIn:`. Then the call to `selectWith` can use a method reference. The cast to `Predicate2` should also be unnecessary.

Reply:
Hi @donraab , thanks for your comments. I have made the requested changes. Please review and let me know. Thanks.